frontend/src/pulldasher/index.tsx

```diff
@@ -12,7 +12,7 @@ export const Pulldasher: React.FC = function() {
    const pullsCIBlocked = pulls.filter(pull => pull.isCiBlocked());
    const pullsDeployBlocked = pulls.filter(pull => pull.isDeployBlocked());
    const pullsReady = pulls.filter(pull => pull.isReady() && pull.isCiRequired());
-   const pullsDevBlocked = pulls.filter(pull => pull.getDevBlock());
+   const pullsDevBlocked = pulls.filter(pull => pull.getDevBlock() && pull.isDraft());
    const pullsNeedingCR = pulls.filter(pull => !pull.isCrDone() && !pull.getDevBlock());
    const pullsNeedingQA = pulls.filter(pull => !pull.isQaDone() && !pull.getDevBlock() && pull.hasPassedCI());
```

Comment:
I think you'll need to add a `!isDraft` check to the CR and QA pulls lists:
```suggestion
   const pullsNeedingCR = pulls.filter(pull => !pull.isCrDone() && !pull.getDevBlock() && !pull.isDraft());
   const pullsNeedingQA = pulls.filter(pull => !pull.isQaDone() && !pull.getDevBlock() && pull.hasPassedCI() && !pull.isDraft());
```

Reply:
Resolved via https://github.com/iFixit/pulldasher/pull/329/commits/79bc61452c623139e34e13c4a9f006b8144c2283